### ISSUE TITLE
Configuring LogConsistencyProvider still requires legacy config in 2.0-rc2

### ISSUE THREAD

**Issue by jms69** [dotnet/orleans#4286]
To configure the EventSourcing I can only use legacy config.

Will this be added to 2.0 config by release?

```
var config = new ClusterConfiguration();
config.AddLogStorageBasedLogConsistencyProvider("StateStorage");  

builder.AddLegacyClusterConfigurationSupport(config);
```

**Comment by jason-bragg** [comment]
Good catch.  Thanks.  Looking into.

**Comment by jason-bragg** [comment]
Addressed in  #4292
